doc/src/records/dhwsys.md

```diff
@@ -144,26 +155,149 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
   ---------- ---------------------- ---------------------- -------------- -----------------
               *name of a DHWDAYUSE*   (no scheduled draws)        No             daily
 
-**wsFaucetWaste=*float***\
-**wsShowerWaste=*float***\
-**wsBathWaste=*float***\
-**wsCWashrWaste=*float***\
-**wsDWashrWaste=*float***
+**wsFaucetDrawDurF=*float***
 
-Specifies additional draw volume per DHWUSE event (at fixture, by end use).  This can be used to account for water discarded during warmup or otherwise adjust the draw volume.  Because the values are at the fixture, the impact on hot water demand additionally depends on DHWUSE parameters.  The value is applied by lengthening (or shortening) the draw duration.
+Draw duration factor for faucets. This can be used to account for water discarded during warmup or otherwise adjust the draw volume.  Because the values are at the fixture, the impact on hot water demand additionally depends on DHWUSE parameters.  The value is applied by lengthening (or shortening) the draw duration.
```

Comment:
It looks like the content for `ws*Waste` was copied to `ws*DrawDurF` by accident.